src/applications/vaos/utils/selectors.js

```diff
@@ -57,7 +63,12 @@ export function getTypeOfCare(data) {
 export function getCCEType(state) {
   const data = getFormData(state);
 
-  const typeOfCare = TYPES_OF_CARE.find(care => care.id === data.typeOfCareId);
+  let typeOfCare = TYPES_OF_CARE.find(care => care.id === data.typeOfCareId);
+  if (typeOfCare.id === 'EYE') {
```

Comment:
This could probably use the const defined on line 41 instead of a string literal